resolwe_bio/tests/processes/test_annotation.py

```diff
@@ -0,0 +1,158 @@
+                    "species": "Homo sapiens",
+                    "build": "GRCh38",
+                },
+            )
+
+        variant_annotation = self.run_process(
+            "variant-annotation",
+            {
+                "database": "GRCh38.109",
+                "dbsnp": dbsnp.id,
+                "clinvar": clinvar.id,
+                "canonical_transcripts": canonical_transcripts.id,
+                "species": "Homo sapiens",
+                "genome_assembly": "GRCh38",
+            },
+            Data.STATUS_ERROR,
+        )
+
+        error_msg = ["No variants found in the database for the selected inputs."]
+        self.assertEqual(variant_annotation.process_error, error_msg)
+
+        variant = Variant.objects.get_or_create(
+            species="Homo sapiens",
+            genome_assembly="GRCh38",
+            chromosome="12",
+            position=25206394,
+            reference="A",
+            alternative="T",
+        )
+        VariantCall.objects.get_or_create(
+            sample=self.sample,
+            variant=variant[0],
+            quality=70,
+            depth_norm_quality=0.7,
+            alternative_allele_depth=11,
+            depth=30,
+            filter="PASS",
+            genotype="0/1",
+            genotype_quality=99,
+        )
+
+        variant_annotation = self.run_process(
+            "variant-annotation",
+            {
+                "database": "GRCh38.109",
+                "dbsnp": dbsnp.id,
+                "clinvar": clinvar.id,
+                "canonical_transcripts": canonical_transcripts.id,
+                "species": "Homo sapiens",
+                "genome_assembly": "GRCh38",
+                "advanced": {"save_vcf": True},
+            },
+        )
+
+        expected_annotation = {
+            "id": variant[0].annotation.id,
+            "variant_id": variant[0].id,
+            "type": "SNP",
+            "clinical_diagnosis": "Noonan_syndrome",
+            "clinical_significance": "Likely_benign",
+            "dbsnp_id": "rs1137189",
+            "clinvar_id": "308075",
+            "transcripts": [
+                {
+                    "id": variant[0].annotation.transcripts.first().id,
+                    "variant_annotation_id": variant[0].annotation.pk,
+                    "annotation": "3_prime_UTR_variant",
+                    "annotation_impact": "MODIFIER",
+                    "gene": "KRAS",
+                    "protein_impact": "",
+                    "transcript_ids": ["ENST00000256078"],
+                    "canonical": False,
+                },
+            ],
+        }
+
+        self.assertDictEqual(
+            VariantAnnotationSerializer(variant[0].annotation).data["transcripts"][0],
+            expected_annotation["transcripts"][0],
+        )
+        self.assertFile(
+            variant_annotation, "vcf", output_folder / "annotated_variants.vcf.gz"
+        )
+
+        variant_annotation = self.run_process(
```

Comment:
Should be followed by an `assert`.

Reply:
Will add once we resolve the errors on the backend.